android/build.gradle

```diff
@@ -67,6 +66,12 @@ task javadocs(type: Javadoc) {
     }
 }
 
+afterEvaluate {
```

Comment:
Instead of afterEvaluate, is it possible to wrap the `android.libraryVariants.collect...` in a closure (`{}`) and then add it within the `javadocs` task? Closures are lazily computed, so it seems like that may work.

afterEvaluate should be avoided when possible, and it is almost always possible to avoid it.

Reply:
Does this look better now? 

Thanks for @dapengzhang0 's suggestion.

Reply:
Eventually much better.